src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.hpp

```diff
@@ -32,12 +32,19 @@ class LLMInferRequest final : public ov::ISyncInferRequest {
 
 private:
     void prepare_for_new_conversation();
+
+    void clear_chunk_prefill_kv_cache();
+
     void init_tensor(const ov::Output<const ov::Node>& port);
 
     void infer_prefill(ov::SoPtr<ov::ITensor> input_ids,
                        ov::SoPtr<ov::ITensor> attention_mask,
                        ov::SoPtr<ov::ITensor> position_ids);
 
+    void infer_prefill_in_chunk(ov::SoPtr<ov::ITensor> input_ids,
+                       ov::SoPtr<ov::ITensor> attention_mask,
```

Comment:
I think indent is broken here

Reply:
this is what clang-format fixes for free.